Gems/GraphCanvas/Code/Source/Components/NodePropertyDisplays/VectorNodePropertyDisplay.cpp

```diff
@@ -302,13 +358,27 @@ namespace GraphCanvas
         {
             m_proxyWidget = new QGraphicsProxyWidget();
 
+            QWidget* widgetContainer  = new QWidget(nullptr);
+            QHBoxLayout* layout = new QHBoxLayout(widgetContainer);
+            widgetContainer->setSizePolicy(QSizePolicy::MinimumExpanding, QSizePolicy::Fixed);
+            
+            layout->setAlignment(Qt::AlignLeft);
+            layout->setContentsMargins(0, 0, 0, 0);
+
+            m_button = new QToolButton();
+            m_button->setSizePolicy(QSizePolicy::Fixed, QSizePolicy::Fixed);
```

Comment:
Instead, you can call setFixedSize and set it to whatever height and width match the vector text fields height.